extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -497,6 +497,12 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     }
     this.fireDelayedImpressions(responseHeaders.get('X-AmpImps'));
     this.fireDelayedImpressions(responseHeaders.get('X-AmpRSImps'), true);
+
+    const refreshInterval = Number(responseHeaders.get('amp-force-refresh'));
+    if (refreshInterval) {
```

Comment:
Assuming you use the change I suggest before for initializeRefreshManagerIfEligible_, this can just be:

this.initializeRefreshManagerIfEligible_(Number(responseHeaders.get('amp-force-refresh')));